### ISSUE TITLE
Featured News Block

### ISSUE THREAD

**Issue by spelkey-ucd** [UCDavisLibrary/ucdlib-theme-wp#152]
http://dev.webstyleguide.ucdavis.edu/redesign/?p=molecules-featured-article

**Comment by wrenaria** [comment]
Enhancement request for this block:

Can the Featured News Article block pull in the latest news article by default with an option to override with a custom selection so that it's not a manual process to update the featured article every time a news post is added? Most of the time, Jessica just wants the latest thing showing there. 

(Low priority, can wait until post-launch)

**Comment by spelkey-ucd** [comment]
Right, to highlight a specific post, you would remove the query block, and put in the original Featured News Block.

**Comment by wrenaria** [comment]
Oh, I see. That's not super intuitive / going to be hard to remember as an occasional use item. Is it not possible to handle it in a single block?

**Comment by spelkey-ucd** [comment]
Sure it's possible, but I would say not advisable. It would be a lot of special handling for one little thing (none of the other cards have this feature).

**Comment by wrenaria** [comment]
Got it. We will just need to have good easy to find documentation on the intranet for those who need to know how to do this. Thanks Steve!
